CHANGES.rst

```diff
@@ -3,10 +3,6 @@ Version 0.2.0
 
 Unreleased
 
--   :class:`cachelib.Simplecache` and :class:`cachelib.FileSystemCache` will
```

Comment:
Did you mean to remove this changelog?

Reply:
Yes, that was intentional. This change will only be applied in #26 so I'll move the changelog update there. This is a lingering change of #13 that should not have been merged. This PR will be merged before #26 since I still need to do some adjustments over there. I can, of course, open a separate PR for this if prefer.